Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -17,7 +17,36 @@ class ContentBlockerHelper {
     fileprivate weak var tab: Tab?
     fileprivate weak var profile: Profile?
 
-    static var blockImagesRule: WKContentRuleList?
+    static fileprivate var blockImagesRule: WKContentRuleList?
+    static fileprivate var whitelistedDomains = [String]()
+
+    // Only set and used in UI test
+    static weak var testInstance: ContentBlockerHelper?
+
+    func whitelist(enable: Bool, forDomain domain: String, completion: (() -> Void)?) {
+        if enable {
+            ContentBlockerHelper.whitelistedDomains.append(domain)
+        } else {
+            ContentBlockerHelper.whitelistedDomains = ContentBlockerHelper.whitelistedDomains.filter { $0 != domain }
+        }
+
+        removeAllRulesInStore {
+            self.compileListsNotInStore {
+                NotificationCenter.default.post(name: .ContentBlockerUpdateNeeded, object: nil)
+                completion?()
+            }
+        }
+
+        if let dir = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first {
```

Comment:
Should this be a `guard` that logs to Sentry if it can't get the dir? I know that probably shouldn't ever happen, but maybe it can?

Reply:
Never hurts to log more errors, I'll add that